eva-server/src/test/resources/test-data/format_jsons_4_testdata.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+import sys
+
+# Script used to convert a file with one json per line,
+# to a file suitable for Lord of the Jars to use as a test
+# dataset to load to a testing mongo database
```

Comment:
Please review capitalization (JSON, Mongo...), and clarify why would the test dataset not be suitable otherwise.